go.mod

```diff
@@ -12,3 +12,5 @@ require (
 	github.com/sapcc/gophercloud-sapcc v0.0.0-20210519124902-12e14f430faf
 	github.com/sapcc/limes v0.0.0-20210730100523-f761e2bd83bc
 )
+
+replace github.com/sapcc/gophercloud-sapcc => ../gophercloud-sapcc
```

Comment:
You probably don't want to merge this.

Reply:
Yeah, I have that temporarily there to work with my local changes. I'll remove this before making the PR _Ready for review_.